src/main/java/teammates/common/datatransfer/attributes/FeedbackResponseCommentAttributes.java

```diff
@@ -39,7 +39,7 @@
     public Text commentText;
     public String lastEditorEmail;
     public Date lastEditedAt;
-    private Long feedbackResponseCommentId;
+    public Long feedbackResponseCommentId;
```

Comment:
A quick comment:
This field should remain as `private`. You could employee another method to compare comment. Maybe for test data, comment content is unique, thus, we may just need to compare the content rather than the whole comment attribute.